### ISSUE TITLE
Khmer font is not working on all machines

### ISSUE THREAD

**Issue by WorldsEndless** [BYU-ODH/cambodian-child-theme#33]
My browsers can't render the font, neither Chrome nor Firefox. The font should be brought in as a distinct solution, not dependent on a system having it installed before visiting. 

![cohp-font](https://user-images.githubusercontent.com/5042778/127226593-db9376ac-2a4c-4a3d-bef4-24e299d3d7f5.png)

**Comment by briancroxall** [comment]
I'm going to close this as it's only been reported by one user as a problem.

**Comment by WorldsEndless** [comment]
The problem is that some Linux systems do not have the Leelawadee font that that is needed to render khmei characters. The font in included in Windows and presumedly mac installs, but must be acquired deliberately for the  site to render on machines that do not have any font that renders the glyphs. I fixed this by downloading and installing the font at http://legionfonts.com/fonts/leelawadee-ui , which was a matter of simply  putting the ttf file in my `~/.fonts` directory.

In case it is gone from the legionfonts site, here is the .ttf I installed. 
[Leelawadee ui.zip](https://github.com/BYU-ODH/cambodian-child-theme/files/13126816/Leelawadee.ui.zip)
